client/src/components/rounds/Apply.tsx

```diff
@@ -40,6 +40,8 @@ function Apply() {
     const roundError = roundState ? roundState.error : undefined;
     const round = roundState ? roundState.round : undefined;
 
+    console.log("DASA round", round);
```

Comment:
I think we can remove this

Reply:
Yep.

I added a local pre-commit hook to check for leftover console.log(s) in my changes. Feel free to steal it from [here](https://gist.github.com/DanieleSalatti/13b2a516ee499bb701ee90a67ab8a553).